.gitignore

```diff
@@ -6,6 +6,8 @@
 /build
 /captures
 .externalNativeBuild
+/common
+/common-jvm
```

Comment:
Is this intentional? If not, please revet them.

Reply:
this is intentional.
When I forked first time, two directories 'common' and 'common-jvm' are still there. But now, I can't find those directory in your repository. So, I add it.

As you can see the Fork Graph, my repository fell down the chaos. I couldn't my repository be cleared. 
So "merge commits" did many times, and add "almost not there directory" to gitignore.
I wanna repair(resolve?) this problem, but I couldn't. please understand :)

Reply:
I see.
but I think these should not be added to .gitignore. because they contains model classes and  should be managed by git. I'm not sure but maybe your issue would be resolved by re-fetching master branch.

What do you think? @takahirom

Reply:
I agree with konifar. Probably that empty folder is not harmful. 
If you fail to fetch you can delete the folder manually. 👍

Reply:
Okay, I'll revert .gitignore. plz wait...